ITE/1/README.adoc

```diff
@@ -0,0 +1,925 @@
+= ITE-1: in-toto Enhancement Format
+:source-highlighter: pygments
+:toc: preamble
+:toclevels: 2
+ifdef::env-github[]
+:tip-caption: :bulb:
+:note-caption: :information_source:
+:important-caption: :heavy_exclamation_mark:
+:caution-caption: :fire:
+:warning-caption: :warning:
+endif::[]
+
+.Metadata
+[cols="2"]
+|===
+| ITE
+| 1
+
+| Title
+| in-toto Enhancement format
+
+| Sponsor
+| link:https://github.com/santiagotorres[Santiago Torres-Arias]
+
+| Status
+| Active :smile:
+
+| Type
+| Process
+
+| Created
+| 2018-10-04
+
+|===
+
+
+[[abstract]]
+== Abstract
+
+An in-toto Enhancement (ITE) is a design document that describes a new feature
+or aspect of the in-toto framework, a setup of its supply chain or an,
```

Comment:
Please consider taking a look at my proposed fixes (https://github.com/lukpueh/ITE/compare/jep-xxxx...lukpueh:jep-xxxx-lp).

Above issue, for instance, is already fixed.

Reply:
Oh, I wasn't aware of that. I only saw your comments directly on this page.
Is there a direct link from this page to all proposed fixes, such as yours? (otherwise, how can I see all proposed fixes?)

Reply:
See my review summary above, i.e. https://github.com/in-toto/ITE/pull/1#pullrequestreview-162349806